src/comms.cpp

```diff
@@ -232,9 +233,20 @@ void comms_loop() {
     }
 }
 
+// Check if it's been long enough to turn on the LED
+void led_loop() {
+    if (digitalRead(LED_BUILTIN) && millis() > garage_door.led_on_time) {
+        digitalWrite(LED_BUILTIN, LOW);
+    }
+}
+
```

Comment:
I added a late comment on your other commit.  I was thinking maybe we could change this to a serviceTimerLoop() and move the code from https://github.com/ratgdo/homekit-ratgdo/blob/v0.6.0/src/comms.cpp#L217 here

Reply:
consider putting this in ratgdo.cpp since it's not related to transmission or receipt of data _per se_